arm_gdb/common.py

```diff
@@ -30,6 +30,8 @@ def read_reg(inf, addr, len):
     bs = inf.read_memory(addr, len).tobytes()
     return sum(v << (8*i) for i, v in enumerate(bs))
 
+def write_reg(inf, addr, buf, len):
+    inf.write_memory(addr, buf, len)
```

Comment:
I would expect the API to this to have same types as `read_reg()` of all arguments/return values, except that `value` would be  an argument for `write_reg()` that matches the return value of `read_reg()`. So `buf` should not be `bytes` or `bytearray` when reg values are `int`

This should work to read and write back the same value:

```
value = read_reg(inf, addr, len)
write_reg(inf, addr, value, len)
```

Reply:
done.